week-3/reading-list/script.js

```diff
@@ -21,3 +21,41 @@ const books = [
   },
 ];
 
+function createBookElement(book) {
+  const bookElement = document.createElement("li");
+  bookElement.style.backgroundColor = book.alreadyRead ? "green" : "red"; //
+  bookElement.style.marginBottom = "5rem";
+
+  const titleElement = document.createElement("span");
+  titleElement.textContent = book.title;
+  bookElement.appendChild(titleElement);
+
+  const authorElement = document.createElement("p");
+  authorElement.textContent = book.author;
+  bookElement.appendChild(authorElement);
+
+  const imageElement = document.createElement("img");
+  imageElement.src = book.bookCoverImage;
+  imageElement.alt = `${book.title} Cover`;
+  bookElement.appendChild(imageElement);
+
+  return bookElement;
+}
+
+function readingList(books) {
```

Comment:
One naming recommendation. Functions should describe action. This allow us to distinguish functions and variables just even by names. I would suggest rename it to `buildReadingList` or `renderReadingList`.